### ISSUE TITLE
Missing WebMemoryBar icon

### ISSUE THREAD

**Issue by mgarin** [mgarin/weblaf#608]
Reported by @kovadam69 on Gitter:
> I discovered an exception using the InterfaceInspector:
```
Exception in thread "AWT-EventQueue-0" com.alee.managers.style.StyleException: Unable to find component icon: class com.alee.extended.memorybar.WebMemoryBar
    at com.alee.managers.style.AbstractComponentDescriptor.getIcon(AbstractComponentDescriptor.java:244)
    at com.alee.extended.inspector.info.WComponentPreview.getIconImpl(WComponentPreview.java:51)
    at com.alee.extended.inspector.info.WComponentPreview.getIconImpl(WComponentPreview.java:37)
    at com.alee.extended.inspector.info.AbstractComponentPreview.getIcon(AbstractComponentPreview.java:74)
    at com.alee.extended.inspector.InterfaceTreeNode.getIcon(InterfaceTreeNode.java:118)
    at com.alee.extended.inspector.InterfaceTreeNode.getIcon(InterfaceTreeNode.java:53)
    at com.alee.laf.tree.WebTreeCellRenderer.iconForValue(WebTreeCellRenderer.java:262)
    at com.alee.laf.tree.WebTreeCellRenderer.updateView(WebTreeCellRenderer.java:179)
    at com.alee.laf.tree.WebTreeCellRenderer.getTreeCellRendererComponent(WebTreeCellRenderer.java:340)
    at javax.swing.plaf.basic.BasicTreeUI$NodeDimensionsHandler.getNodeDimensions(BasicTreeUI.java:2807)
    at javax.swing.tree.AbstractLayoutCache.getNodeDimensions(AbstractLayoutCache.java:492)
    at javax.swing.tree.VariableHeightLayoutCache$TreeStateNode.updatePreferredSize(VariableHeightLayoutCache.java:1360)
    at javax.swing.tree.VariableHeightLayoutCache$TreeStateNode.getXOrigin(VariableHeightLayoutCache.java:1143)
    at javax.swing.tree.VariableHeightLayoutCache$TreeStateNode.getNodeBounds(VariableHeightLayoutCache.java:1130)
    at javax.swing.tree.VariableHeightLayoutCache.getBounds(VariableHeightLayoutCache.java:220)
    at javax.swing.plaf.basic.BasicTreeUI.getPathBounds(BasicTreeUI.java:529)
    at javax.swing.plaf.basic.BasicTreeUI.getPathBounds(BasicTreeUI.java:522)
    at com.alee.laf.tree.WebTreeUI.getFullPathBounds(WebTreeUI.java:251)
    at com.alee.laf.tree.WebTreeUI.getPathBounds(WebTreeUI.java:239)
    at com.alee.laf.tree.WebTreeUI.getExactRowForLocation(WebTreeUI.java:197)
    at com.alee.laf.tree.WebTreeUI.getExactRowForLocation(WebTreeUI.java:182)
    at com.alee.laf.tree.behavior.TreePathHoverBehavior.getObjectAt(TreePathHoverBehavior.java:69)
    at com.alee.laf.tree.behavior.TreePathHoverBehavior.getObjectAt(TreePathHoverBehavior.java:37)
    at com.alee.extended.behavior.AbstractObjectHoverBehavior.updateHover(AbstractObjectHoverBehavior.java:379)
    at com.alee.extended.behavior.AbstractObjectHoverBehavior.updateHover(AbstractObjectHoverBehavior.java:366)
    at com.alee.extended.behavior.AbstractObjectHoverBehavior.mouseMoved(AbstractObjectHoverBehavior.java:171)
    at java.awt.AWTEventMulticaster.mouseMoved(AWTEventMulticaster.java:330)
    at java.awt.AWTEventMulticaster.mouseMoved(AWTEventMulticaster.java:329)
    at java.awt.Component.processMouseMotionEvent(Component.java:6578)
    at javax.swing.JComponent.processMouseMotionEvent(JComponent.java:3342)
    at java.awt.Component.processEvent(Component.java:6302)
    at java.awt.Container.processEvent(Container.java:2238)
    at java.awt.Component.dispatchEventImpl(Component.java:4889)
    at java.awt.Container.dispatchEventImpl(Container.java:2296)
    at java.awt.Component.dispatchEvent(Component.java:4711)
    at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4897)
    at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
    at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4475)
    at java.awt.Container.dispatchEventImpl(Container.java:2282)
    at java.awt.Window.dispatchEventImpl(Window.java:2746)
    at java.awt.Component.dispatchEvent(Component.java:4711)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:760)
    at java.awt.EventQueue.access$500(EventQueue.java:97)
    at java.awt.EventQueue$3.run(EventQueue.java:709)
    at java.awt.EventQueue$3.run(EventQueue.java:703)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
```
> Seems some icon is missing for WebMemoryBar.

**Comment by mgarin** [comment]
Added the missing icon, it will be available in v1.3.0 release.

**Comment by mgarin** [comment]
Just a small note - this will be available in v1.2.13 update that will be going live shortly. 
I will take some more time to polish v1.3.0 and meanwhile will release a few smaller updates.
